docs/inventory.md

```diff
@@ -132,7 +141,11 @@ Now you can set the path of the ansible inventory in the source:
 
 ### <a name='source-netbox'></a>Netbox
 
-[Netbox](https://netbox.readthedocs.io/en/stable/) is often used to store devices type, management address and other useful information to be used in network automation. Suzieq can pull device data from Netbox selecting them by tag (currently only one per each source). To do so a token to access the netbox API is required as well as the netbox instance url.
+[Netbox](https://netbox.readthedocs.io/en/stable/) is often used to store devices type, management address and other useful information to be used in network automation. Suzieq can pull device data from Netbox selecting them by tag (currently only one per each source).
+To do so a token to access the netbox API is required as well as the netbox instance url.
+
+The token is considered a [sensitive data](#sensitive-data), so it can be specified via an environment variable using the format `env:ENV_TOKEN`.
```

Comment:
Just to be consistent
```suggestion
The token is considered a [sensitive data](#sensitive-data), so it can be specified via an environment variable using the format `env:<ENV_TOKEN>`.
```